cogboard-webapp/src/components/Widget/index.js

```diff
@@ -164,7 +164,9 @@ const Widget = ({ id, index }) => {
         type={type}
         expanded="true"
       >
-        {(isAuthenticated || widgetStatus !== 'NONE' || title !== '') && (
+        {(isAuthenticated ||
+          widgetStatus !== 'NONE' ||
+          type === 'TextWidget') && (
```

Comment:
I think it would be better if widget component won't be aware of literal widget types.